src/test/java/nextstep/subway/line/LineAcceptanceTest.java

```diff
@@ -96,18 +38,27 @@ void createLine() {
         LineResponse lineResponse = response.response().getBody().as(LineResponse.class);
         Assertions.assertThat(lineResponse.getName()).isEqualTo("분당선");
         Assertions.assertThat(lineResponse.getColor()).isEqualTo("yellow");
+        Assertions.assertThat(lineResponse.getStations().get(0).getName()).isEqualTo("강남역");
+        Assertions.assertThat(lineResponse.getStations().get(1).getName()).isEqualTo("신도림역");
```

Comment:
아래와 같이 assertj 검증문을 활용해도 좋을 것 같아요~!

```suggestion
        Assertions.assertThat(lineResponse.getStations())
                .extracting(StationResponse::getName)
                .containsExactly("강남역", "신도림역");
```